sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1657,3 +1657,146 @@ case class MakeDate(year: Expression, month: Expression, day: Expression)
 
   override def prettyName: String = "make_date"
 }
+
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = "_FUNC_(year, month, day, hour, min, sec, timezone) - Create timestamp from year, month, day, hour, min, sec and timezone fields.",
+  arguments = """
+    Arguments:
+      * year - the year to represent, from 1 to 9999
+      * month - the month-of-year to represent, from 1 (January) to 12 (December)
+      * day - the day-of-month to represent, from 1 to 31
+      * hour - the hour-of-day to represent, from 0 to 23
+      * minute - the minute-of-hour to represent, from 0 to 59
+      * second - the second-of-minute and its micro-fraction to represent, from
+                 0 to 60. If the second argument equals to 60, the seconds field is set
+                 to 0 and 1 minute is added to the final timestamp.
+      * timezone - optional time zone identifier. For example, CET, UTC and etc.
```

Comment:
Please remove `optional` because optionality is a kind of syntax instead of parameter description.